homeassistant/components/icloud/config_flow.py

```diff
@@ -64,6 +66,10 @@ def __init__(self):
                     vol.Required(
                         CONF_PASSWORD, default=user_input.get(CONF_PASSWORD, "")
                     ): str,
+                    vol.Required(
```

Comment:
IMHO we should stay consistent with init.py and therefore make it optional.

Reply:
So `Optional` will it be 😉